gig/helpers.py

```diff
@@ -64,14 +68,48 @@ def update_plan_default_section(assoc):
     """
     Plan.objects.filter(assoc=assoc, plan_section=None).update(section=assoc.default_section)
 
+def date_format_func(dt, fmt):
+    # Returns lambdas that the template will evaluate in the correct
+    # language context
+    return lambda: date_format(template_localtime(dt), fmt)
+
+def date_diff(latest, previous):
+    if latest.date() == previous.date():
```

Comment:
There might not be a previous date - enddate is 'none' by default, and if you add one later this fails

Reply:
Good catch!  This is fixed, and tested, now.